worker.Dockerfile

```diff
@@ -1,10 +1,29 @@
-FROM jjanzic/docker-python3-opencv
+# FROM jjanzic/docker-python3-opencv
+FROM nvidia/cuda:11.0-runtime-ubuntu18.04
+
+RUN apt-get update
+RUN apt-get install python3.6 -y
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install python3-pip -y 
 
 ARG FLASK_ENV=production
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
 WORKDIR /app
 COPY . /app
-RUN pip install --upgrade cython && pip install -r worker.requirements.txt && python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'
-CMD python worker.py
+
+RUN pip3 install --upgrade pip
+RUN pip3 install PyYAML
+
+RUN pip3 install numpy
```

Comment:
Why are these happening here and not in `requirements.txt`?

Reply:
The file now installs numpy, cython, and then requirements.txt in that order because cython requires numpy as a dependency, and cython bbox in the requirements file requires cython as a dependency